### DISCUSSION TITLE
Debugging Grunge Stack

### DISCUSSION THREAD

**Discussion by gnllucena** [remix-run/remix#4097] [Q&A]
Hey, guys!

I'm trying to debug grunge-stack loader and action functions within vscode, I've successfully debug blues-stack and indie-stack just using an plain launch.json settings:

```json
{
  "version": "0.2.0",
  "configurations": [
    {
      "command": "npm run dev",
      "name": "Run npm run dev",
      "request": "launch",
      "type": "node-terminal",
      "cwd": "${workspaceFolder}"
    },
  ]
}
```

I've seen that @tbeseda (https://github.com/remix-run/grunge-stack/issues/25#issuecomment-1073966536) gives a good explanation to why this happens, but it seems to me that choosing a stack that isn't debuggable nowadays is not a good decision.

Do you guys know any workaround this behavior? Is there anything I can do to debug grunge-stack?

**Comment by coanjos** [comment]
I was having the same behavior here. Watching this issue for any future answers.

**Comment by quantuminformation** [comment]
Same issue here, I tried 
```json
      "command": "npm run dev",
      "name": "Run npm run dev",
      "request": "launch",
      "type": "node-terminal",
      "cwd": "${workspaceFolder}"
      ```
its prob something special about the lambda mocking, so prob not really a remix issue - what did the lambda docs say about debugging?
